db/data/20210412134546_fix_extending_organisation_inconsistencies.rb

```diff
@@ -0,0 +1,18 @@
+# Run me with `rails runner db/data/20210412134546_fix_extending_organisation_inconsistencies.rb`
+
+beis = Organisation.find_by(service_owner: true)
+
+Activity.fund.update_all(organisation_id: beis.id, extending_organisation_id: beis.id)
+
+Activity.programme.where.not(extending_organisation_id: beis.id).update_all(organisation_id: beis.id)
+
+Activity.project.update_all("extending_organisation_id = organisation_id")
+
+Activity.third_party_project.update_all("extending_organisation_id = organisation_id")
+
+bad_programmes = Activity.programme.where(extending_organisation_id: beis.id)
+
+if bad_programmes.any?
+  puts "Found #{bad_programmes.size} programme(s) whose extending_organisation is incorrectly set to BEIS and should be manually corrected:"
+  puts bad_programmes.ids.join("\n")
```

Comment:
How would you feel about generating the list of "bad" programmes as a list of production URLs? ("Nope" is a valid answer.)

Reply:
Nice idea, but the Activity details page is scoped on organisation, and this warning is flagging places where it is unreliable.